tutorials/introductory/customizing.py

```diff
@@ -1,32 +1,113 @@
 """
 .. redirect-from:: /users/customizing
 
+=====================================================
 Customizing Matplotlib with style sheets and rcParams
 =====================================================
 
 Tips for customizing the properties and default styles of Matplotlib.
 
-Using style sheets
-------------------
+There are three ways to customize Matplotlib:
+
+    1. :ref:`Setting rcParams at runtime<customizing-with-dynamic-rc-settings>`.
+    2. :ref:`Using style sheets<customizing-with-style-sheets>`.
+    3. :ref:`Changing your matplotlibrc file<customizing-with-matplotlibrc-files>`.
+
+Setting rcParams at runtime takes precedence over style sheets, style
+sheets take precedence over :file:`matplotlibrc` files.
+
+.. _customizing-with-dynamic-rc-settings:
 
-The :mod:`.style` package adds support for easy-to-switch plotting
-"styles" with the same parameters as a :ref:`matplotlib rc
-<customizing-with-matplotlibrc-files>` file (which is read at startup to
-configure Matplotlib).
+Dynamic rc settings
```

Comment:
I think "dynamic" is too technical / jargon for the average user. Can we go with simply "Runtime rc settings"?

Reply:
Maybe both?  "Runtime" is definitely jargon, though less vague than "dynamic".

Reply:
Sure, I'm also more happy with "runtime" than with "dynamic".

I only changed the heading now and kept the internal references.

Reply:
Ok, than I was a bit too fast. Just tell me what you would like the heading to be and I will change it accordingly.

Reply:
I wouldn't over-think this. IMHO "runtime" is good enough. I've just taking the same wording as in the the enumeration above: "Setting rcParams at runtime". It makes sense that the headings reflects this enumeration.

On a more general note, the whole naming, wording and structure around config is somewhat odd. But that's something we won't be able to solve in this PR. I plan to bring this up for 3.6.